hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/HBaseInterClusterReplicationEndpoint.java

```diff
@@ -113,6 +113,15 @@
   private boolean dropOnDeletedTables;
   private boolean isSerial = false;
 
+  protected AsyncClusterConnection createConnection(Configuration conf) throws IOException {
```

Comment:
nit: Add a quick comment why these exist as protected? Otherwise it can confuse the readers because the only callers in the source are in this class.